src/routes.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+use Io\Token\Proto\Common\Alias\Alias;
+use Tokenio\Config\TokenCluster;
+use Tokenio\Config\TokenEnvironment;
+use Tokenio\Config\TokenIoBuilder;
+use Tokenio\Http\Request\TokenRequest;
+use Tokenio\Http\Request\TokenRequestOptions;
+use Tokenio\Security\UnsecuredFileSystemKeyStore;
+
+class TokenSample
+{
+    const DEVELOPER_KEY = '4qY7lqQw8NOl9gng0ZHgT4xdiDqxqoGVutuZwrUYQsI';
+
+    /**
+     * @var UnsecuredFileSystemKeyStore
+     */
+    private $keyStore;
+
+    private $tokenIO;
+    private $member;
+
+    public function __construct()
+    {
+        $this->keyStore = new UnsecuredFileSystemKeyStore(__DIR__ . '/../keys/');
+
+        $this->tokenIO = $this->initializeSDK();
+        $this->member = $this->initializeMember();
+    }
+
+    private function initializeSDK()
+    {
+        $builder = new TokenIoBuilder();
+        $builder->connectTo(TokenCluster::get(TokenEnvironment::SANDBOX));
+        $builder->developerKey(self::DEVELOPER_KEY);
+        $builder->withKeyStore($this->keyStore);
+        return $builder->build();
+    }
+
+    private function initializeMember()
+    {
+        $memberId = $this->keyStore->getFirstMemberId();
```

Comment:
The `getFirstMemberId` looks like a hack to me, the key store isn't supposed to expose its member ids. If it does, it should expose a list of member ids instead of just one. I understand you want to reuse the stored keys. One way, though not elegant, is to read the file names in the key directory, since the file names are just member ids. In real life the user is supposed to store the member ids in a different db, or have their own keystore implementation.

I will remove the `getFirstMemberId` from `UnsecuredFileSystemKeyStore` and publish a new sdk version `0.0.2` shortly. Please update the samples accordingly.